lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -1486,7 +1453,7 @@ input[name="subject"] {
       th {
         text-align: left;
         border-bottom: 1px solid $border-color-1;
-        font-size: 16px;
+        font-size: 13px;
```

Comment:
drive by improvement: those table header fonts were too big and headers were overlapping in an ugly way. Not a regression, but an improvement while i'm in here.